apps/task-app/src/app/components/QueueList.tsx

```diff
@@ -38,5 +38,4 @@ const QueueListComponent: FunctionComponent<QueueListComponentProps> = ({
 export const QueueList = styled(QueueListComponent)`
   padding: 8px 32px 32px 32px;
```

Comment:
Can you change this to use the design token to use --goa-space-xs, --goa-space-xl, --goa-space-xs, --goa-space-xl